go-controller/pkg/ovn/endpoints.go

```diff
@@ -42,7 +42,7 @@ func (ovn *Controller) getLbEndpoints(ep *kapi.Endpoints) map[kapi.Protocol]map[
 }
 
 // AddEndpoints adds endpoints and creates corresponding resources in OVN
-func (ovn *Controller) AddEndpoints(ep *kapi.Endpoints) error {
```

Comment:
One concern I have with this is if we fix our unit testing to actually unit test functions, then we are losing the ability to check the return on this function. I'm not sure what we gain here by not returning the error.

Reply:
Ok, if that is the case then sure, I'll re-add those returns. It's just that it looked silly the way it stood today.